install/eks_eksctl/05_create-secret.md

```diff
@@ -1,6 +1,5 @@
 
 # Secret Yaml Setup
-
```

Comment:
dont delete line

Reply:
Handled in [0ae93f4](https://github.com/ajktown/docs/pull/65/commits/0ae93f4a216a68affc30883ea5728bbc8364bcd2)